import-export-cli/integration/testutils/devFirst_testUtils.go

```diff
@@ -287,6 +287,16 @@ func ValidateImportUpdateProjectNotAlreadyImported(t *testing.T, args *InitTestA
 	})
 }
 
+func ValidateHiddenFilesPresentWithDocs(t *testing.T, TestDestPathForHiddenFiles, TestDestPathForHiddenFilesInRootDir string) {
```

Comment:
```suggestion
func ValidateHiddenFilesPresentWithDocsBeforeImport(t *testing.T, TestDestPathForHiddenFiles, TestDestPathForHiddenFilesInRootDir string) {
```

Shall we rename like this?

Reply:
Fixed via [9ba0bf8](https://github.com/wso2/product-apim-tooling/pull/955/commits/9ba0bf83956c00bce7629f230329a144fc5f7b34)